webrtc/videoroom/assets/src/membraneWebRTC.ts

```diff
@@ -101,12 +101,31 @@ export class MembraneWebRTC {
     this.connection = undefined;
   };
 
+  public muteMicrophone = async () => {
+    this.localTracks.forEach(([track, stream]) => {
+      if (track.kind == "audio") {
+        track.enabled = !track.enabled;
+        this.channel.push("mute-microphone", { data: !track.enabled, mid: this.getSendTrackMid(track) })
+      }
+    });
+    var buttonText = document.getElementById("mute-microphone")?.innerText
+    if (buttonText == "Mute microphone") {
+      document.getElementById("mute-microphone")!.innerText = "Unmute microphone";
+    } else if (buttonText == "Unmute microphone") {
+      document.getElementById("mute-microphone")!.innerText = "Mute microphone";
+    }
+  }
+
+  private getSendTrackMid(track: MediaStreamTrack) {
```

Comment:
```suggestion
  private getSendTrackMid(track: MediaStreamTrack): <here goes return type> {
```

And either change that function to `findTrackTransceiver` and then get `mid` param or rename it to something more descriptive e.g. `retrieveSenderTrackMid`.